app/controllers/topics_controller.rb

```diff
@@ -232,10 +232,10 @@ def posts
     fetch_topic_view(options)
 
     render_json_dump(TopicViewPostsSerializer.new(@topic_view,
-      scope: guardian,
-      root: false,
-      include_raw: !!params[:include_raw]
-    ))
+                                                  scope: guardian,
```

Comment:
for the record I prefer the original format here. One problem with lining up is that if the original line changes in length you have to move all subsequent lines which pollutes the history. This is a bikeshed though.

Reply:
Ah I think my vim autoformatter did this, I agree with you that the original was better